src/other-scripts/recaptcha/index.js

```diff
@@ -118,7 +119,7 @@ function renderWidget( form ) {
 	const options = {
 		sitekey: siteKey,
 		size: isInvisible ? 'invisible' : 'normal',
-		isolated: true, // Seems to avoid a "Cannot contact reCAPTCHA" connection error in some browsers.
+		isolated: true,
```

Comment:
The connection error still happens so removing the comment. I don't think it hurts to keep the option though so didn't remove. Let me know if you think we can get rid of it though!

Reply:
Sounds good to me!